### PR TITLE
@georgejean tweaks for powershell

### PR DESCRIPTION
**Pull request by stonebig**
https://github.com/winpython/winpython/issues/436

### CODE REVIEW

File: make.py [hunk 1]
```diff
@@ -719,12 +720,13 @@ def _create_batch_scripts_initial(self):
         self.create_batch_script('cmd_ps.bat', r"""@echo off
 rem safe bet 
 call "%~dp0env_for_icons.bat"
-Powershell.exe -executionpolicy RemoteSigned -noexit -file "%~dp0WinPython_PS_Prompt.ps1"
+Powershell.exe -Command "& {Start-Process PowerShell.exe -ArgumentList '-ExecutionPolicy RemoteSigned -noexit -File""%~dp0WinPython_PS_Prompt.ps1""'}"
 """)
```
Comment (georgejean) [inline make.py]: +exit

Comment (georgejean) [inline make.py]: II think `exit` must be inserted on this line (line above 724) in order to exit the cmd.exe (the powershell console remains open), see line 729.

Comment (stonebig) [inline make.py]: oups ! I forgot it

Comment (stonebig) [inline make.py]: but it seems not needed

Comment (stonebig) [inline make.py]: maybe you're right... in doubts, I add it

**Pull request merged**